### ISSUE TITLE
[Visual Requirements – Make Code – Home Page] Content present on titles are not properly visible after resizing at 400% under ‘Home Page’.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3522]
**User Experience:**
End user will find difficulty to interact with controls and won't be able to perform functionality over such control if controls are not visible after resizing to 400% due to this, user will be affected.

**Note:** User credentials should NOT be included in the bug.

**Re-pro  Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate to the controls appearing in Home page section.  
4. Go to browser setting page Zoom up to 400%.
5. Verify the issue.
 
**Actual Result:**
Content present on titles are not properly visible after resizing at 400% under ‘Home Page’, due to this, user is unable to interact with controls and won't be able to perform functionality over such controls.
 
**Expected Result:**
Content present on titles should be properly visible after resizing at 400% under ‘Home Page’, so that user could easily interact with controls and could be able to perform functionality over such controls.

**Note:**
Same issue exists on Chrome & Firefox.
Same issue exists throughout of application.

**WCAG Reference:**
[WCAG 1.4.10 - Reflow
](https://www.w3.org/WAI/WCAG21/Understanding/reflow.html)

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG1.4.10, #Win10-Edge(Chromium)

![WCAG1 4 10_Reflow](https://user-images.githubusercontent.com/72871089/97665570-c6a9b680-1aa1-11eb-86fe-6c271953cbd5.jpg)

**Comment by jwunderl** [comment]
When zoomed in we hit the mobile breakpoint which collapses some text to fit these elements on the screen; these titles will be shown in full when the user interacts with the element and expands to the detail view.
